### PR TITLE
Refactor [v105] Rename extensions

### PR DESCRIPTION
**Pull request by thatswinnie**
This renames and cleans ups extensions.

### CODE REVIEW

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -149,7 +149,7 @@
 		28E08C9A1AF44F00009BA2FA /* BrowserSchema.swift in Sources */ = {isa = PBXBuildFile; fileRef = 282915E51AF1A7920006EEB5 /* BrowserSchema.swift */; };
 		28E23C121AC5A5EE00F5AC85 /* State.swift in Sources */ = {isa = PBXBuildFile; fileRef = 28E23C111AC5A5EE00F5AC85 /* State.swift */; };
 		28E91E751B443AD5009DF274 /* SyncConstants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 28E91E741B443AD5009DF274 /* SyncConstants.swift */; };
-		28EADE5D1AFC3A78007FB2FB /* UIImageViewExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 28EADE381AFC3898007FB2FB /* UIImageViewExtensions.swift */; };
+		28EADE5D1AFC3A78007FB2FB /* UIImageView+Utilies.swift in Sources */ = {isa = PBXBuildFile; fileRef = 28EADE381AFC3898007FB2FB /* UIImageView+Utilies.swift */; };
```
Comment (nbhasin2) [inline Client.xcodeproj/project.pbxproj:152]: @thatswinnie  UIImageView+Utilies -> UIImageView+Utils or UIImageView+Utilities as I am not sure if Utilies is a proper name 🤔

Comment (thatswinnie) [reply Client.xcodeproj/project.pbxproj:152]: 😆 `Utils` it is then

Comment (nbhasin2) [reply Client.xcodeproj/project.pbxproj:152]: This is the same theme for all. If anything I'd go with `+ Extensions` or `+ Utils` or + `Utilities`

I feel + Extensions sound a work better as we are dealing with swift extensions and not obj-c categories

Comment (thatswinnie) [reply Client.xcodeproj/project.pbxproj:152]: Ah, saw it too late. I renamed it to `+Utils` already. `+Extensions` would be ok as well. How strongly do you feel about it?

File: Client/Extensions/UIImageView+Utilss.swift [hunk 2]
```diff
@@ -7,15 +7,6 @@ import Storage
 import Shared
 import Kingfisher
```
Comment (lmarceau) [inline Client/Extensions/UIImageView+Utilss.swift:9]: `Client/Extensions/UIImageView+Utilss.swift` -> Has two `s` at the end

**Review by lmarceau — APPROVED** [review]

**Review by nbhasin2 — APPROVED** [review]

**Pull request merged**